docs/general/serializers.md

```diff
@@ -217,7 +217,15 @@ The object being serialized.
 
 #### #root
```

Comment:
lazy ref to discussion of some of the various way root can be configured and what we want to do about it https://github.com/rails-api/active_model_serializers/pull/1794/files